File: FindPlacesNearMe/FindPlacesNearMe/Models/Place+CoreDataProperties.m
//
//  Place+CoreDataProperties.m
//  FindPlacesNearMe
//
//  Created by Mobile OS on 4/3/2020.
//  Copyright © 2020 myproject. All rights reserved.
//
//

#import "Place+CoreDataProperties.h"

@implementation Place (CoreDataProperties)

+ (NSFetchRequest<Place *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Place"];
}

@dynamic distance;
@dynamic latitude;
@dynamic longitude;
@dynamic title;
@dynamic vicinity;
@dynamic sourceSearch;

@end

File: FindPlacesNearMeSwift/FindPlacesNearMeSwift/Model/Place+CoreDataProperties.swift
//
//  Place+CoreDataProperties.swift
//  FindPlacesNearMeSwift
//
//  Copyright © 2020 Amor Rio. All rights reserved.
//

import Foundation
import CoreData


extension Place {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Place> {
        return NSFetchRequest<Place>(entityName: "Place")
    }

    @NSManaged public var distance: Double
    @NSManaged public var latitude: Double
    @NSManaged public var longitude: Double
    @NSManaged public var title: String?
    @NSManaged public var vicinity: String?
    @NSManaged public var sourceSearch: UserLocationSearch?

}
